ws/src/main/scala/me/jeffmay/neo4j/client/rest/package.scala

```diff
@@ -1,8 +1,22 @@
 package me.jeffmay.neo4j.client
 
-import play.api.libs.json.{Json, Reads}
+import me.jeffmay.neo4j.client.cypher._
+import play.api.libs.json.{JsArray, Json, Reads, Writes}
 
 package object rest {
 
   implicit lazy val readsNeo4jError: Reads[Neo4jError] = Json.reads[Neo4jError]
+
+  implicit lazy val writesCypherValue: Writes[CypherValue] = Writes {
+    case CypherArray(vs) => JsArray(vs.map(writesCypherValue.writes))
+    case CypherString(v) => Json.toJson(v)
+    case CypherInt(v) => Json.toJson(v)
+    case CypherDouble(v) => Json.toJson(v)
+    case CypherBoolean(v) => Json.toJson(v)
+    case CypherLong(v) => Json.toJson(v)
+    case CypherFloat(v) => Json.toJson(v)
+    case CypherByte(v) => Json.toJson(new String(Array(v)))
+    case CypherChar(v) => Json.toJson(v.toString)
+    case CypherShort(v) => Json.toJson(v)
+  }
```

Comment:
FAIL: This should have a play-json test spec. Maybe make this a Format[CypherValue] to handle round-trip serialization in the future.